### ISSUE TITLE
Resolve errors reported by Travis Build Config Validation

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#8014]
Identified at https://travis-ci.org/github/checkstyle/checkstyle/builds/670098380/config when clicking "View Config" and expanding the list in "Build config validation":

````
 Build config validation — 6 errors, 2 warnings, 4 infos

jobs.include.env: invalid env var DESC="MacOS JDK13 verify, site ...
jobs.include.env: invalid env var DESC="MacOS JDK11 verify, site ...
jobs.include.env: invalid env var DESC="MacOS JDK8 verify, site, ...
jobs.include.env: invalid env var DESC="MacOS JDK8 verify, site, ...
jobs.include.env: invalid env var DESC="MacOS JDK13 verify, site ...
jobs.include.env: invalid env var DESC="MacOS JDK11 verify, site ...
root: deprecated key sudo (The key `sudo` has no effect anymore.)
root: deprecated key sudo (The key `sudo` has no effect anymore.)
root: key matrix is an alias for jobs, using jobs
root: missing dist, using the default xenial
root: missing os, using the default linux
root: key matrix is an alias for jobs, using jobs
````

We should look into what is being reported and see if we can resolve them.

**Comment by romani** [comment]
Is it final fix ?

**Comment by rnveach** [comment]
Yes, I don't see a way to enforce the build to fail when new errors come up was my only concern.

**Comment by rnveach** [comment]
Closed for now.
